tests/conftest.py

```diff
@@ -28,16 +35,6 @@ def test_spans(tracer):
     container.reset()
 
 
-@pytest.fixture
-def run_python_code_in_subprocess(tmpdir):
-    def _run(code, **kwargs):
-        pyfile = tmpdir.join("test.py")
-        pyfile.write(code)
-        return call_program(sys.executable, str(pyfile), **kwargs)
-
-    yield _run
-
-
 @pytest.fixture
 def ddtrace_run_python_code_in_subprocess(tmpdir):
```

Comment:
Can this also be replaced?

Reply:
this marker now supports `ddtrace-run`, but we can tackle the refactor/updating/removing this fixture in another PR